docker/test/integration/minifi/test/__init__.py

```diff
@@ -40,21 +39,24 @@ def __init__(self, output_validator):
 
         # Create test input/output directories
         test_cluster_id = str(uuid.uuid4())
+        
+        self.segfault = False
 
         self.tmp_test_output_dir = '/tmp/.nifi-test-output.' + test_cluster_id
         self.tmp_test_input_dir = '/tmp/.nifi-test-input.' + test_cluster_id
         self.tmp_test_resources_dir = '/tmp/.nifi-test-resources.' + test_cluster_id
 
         logging.info('Creating tmp test input dir: %s', self.tmp_test_input_dir)
-        os.makedirs(self.tmp_test_input_dir, mode=0777)
+        os.makedirs(self.tmp_test_input_dir)
         logging.info('Creating tmp test output dir: %s', self.tmp_test_output_dir)
-        os.makedirs(self.tmp_test_output_dir, mode=0777)
+        os.makedirs(self.tmp_test_output_dir)
         logging.info('Creating tmp test resource dir: %s', self.tmp_test_resources_dir)
-        os.makedirs(self.tmp_test_resources_dir, mode=0777)
+        os.makedirs(self.tmp_test_resources_dir)
 
         # Point output validator to ephemeral output dir
         self.output_validator = output_validator
-        output_validator.set_output_dir(self.tmp_test_output_dir)
+        if isinstance(output_validator, SingleFileOutputValidator):
+          output_validator.set_output_dir(self.tmp_test_output_dir)
```

Comment:
Indentation should be 4

Reply:
thanks for pointing this out! ! I'll run re-indent.